utils/music_crawlers.py

```
@@ -0,0 +1,847 @@
+                primary_tasks.append(all_crawlers['netease'].search(keyword, limit))
+            else:
+                primary_tasks.append(all_crawlers['soundcloud'].search(keyword, limit))
+                primary_tasks.append(all_crawlers['itunes'].search(keyword, limit))
+                
+        # 执行第一梯队
+        primary_responses = await asyncio.gather(*primary_tasks, return_exceptions=True)
+        for res in primary_responses:
+            if isinstance(res, list) and res:
+                all_results.extend(res)
+                
+        # --- 组建第二梯队（兜底截断逻辑） ---
```

Ruff (0.15.4)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)